core/experiments/manager.php

```diff
@@ -360,44 +376,63 @@ private function register_settings_fields( Settings $settings ) {
 
 			$feature_key = 'experiment-' . $feature_name;
 
-			$fields[ $feature_key ]['label'] = $this->get_feature_settings_label_html( $feature );
+			$section = 'stable' === $feature['release_status'] ? 'stable' : 'ongoing';
 
-			$fields[ $feature_key ]['field_args'] = $feature;
+			$fields[ $section ][ $feature_key ]['label'] = $this->get_feature_settings_label_html( $feature );
 
-			$fields[ $feature_key ]['render'] = function( $feature ) {
+			$fields[ $section ][ $feature_key ]['field_args'] = $feature;
+
+			$fields[ $section ][ $feature_key ]['render'] = function( $feature ) {
 				$this->render_feature_settings_field( $feature );
 			};
 		}
 
-		if ( ! $features ) {
-			$fields['no_features'] = [
-				'label' => esc_html__( 'No available experiments', 'elementor' ),
-				'field_args' => [
-					'type' => 'raw_html',
-					'html' => esc_html__( 'The current version of Elementor doesn\'t have any experimental features . if you\'re feeling curious make sure to come back in future versions.', 'elementor' ),
-				],
-			];
-		}
+		foreach ( [ 'stable', 'ongoing' ] as $section ) {
+			if ( ! $features ) {
```

Comment:
Instead of general $features, change to stable/ongoing.

Reply:
done